### PR TITLE
LPS-178463 Reorganize content for single page design options view

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by sandrodw3** [comment]
ci:test:sf

**Comment by sandrodw3** [comment]
ci:test:sf

**Comment by sandrodw3** [comment]
ci:test:sf

File: modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/layout/look_and_feel/MasterLayoutConfiguration.js [hunk 1]
```diff
@@ -79,14 +79,17 @@ export default function MasterLayoutConfiguration({
...
+				{Liferay.Language.get('master')}
+			</label>
 
 			{editMasterLayoutURL &&
 			masterLayout.plid &&
 			masterLayout.plid !== DEFAULT_MASTER_LAYOUT_PLID ? (
 				<div className="d-flex">
 					<ClayForm.Group className="flex-grow-1 mb-0">
 						<ClayInput
+							id={`${portletNamespace}styleBookEntry`}
```
Comment (victorg1991) [inline modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/layout/look_and_feel/MasterLayoutConfiguration.js:92]: styleBookEntry -> masterLayout ? the one below too

**Comment by victorg1991** [comment]
Does this need to be hidden under a FF?

File: modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/layout/look_and_feel/MasterLayoutConfiguration.js (hunk 1 above)
Comment (sandrodw3) [reply modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/layout/look_and_feel/MasterLayoutConfiguration.js:92]: Ups, thanks 🤍

File: modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/layout/look_and_feel.jsp [hunk 2]
```diff
@@ -259,7 +285,33 @@ else {
...
+		Liferay.Util.toggleDisabled(
+			[
+				regularCss,
+				regularCssLabel,
+				themeCssExtension,
+				themeCssExtensionLabel,
+				themeCssExtensionSelect,
+			],
+			false
+		);
```
[the remark this answers is not in the document]
Comment (sandrodw3) [reply modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/layout/look_and_feel.jsp]: Done, very much cleaner to be honest 😄 thanks!

**Comment by victorg1991** [comment]
ci:forward

**Comment by victorg1991** [comment]
LGTM :D 👏

**Comment by liulukiwi** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/11598

**Pull request closed without merging**